ssl/quic/quic_impl.c

```diff
@@ -2577,6 +2580,10 @@ static int quic_write_nonblocking_epw(QCTX *ctx, const void *buf, size_t len,
 
     quic_post_write(xso, *written > 0, *written == len, flags,
                     qctx_should_autotick(ctx));
+    if (*written == 0)
+        /* SSL_write_ex returns 0 if it didn't read anything .*/
```

Comment:
this looks odd to me. comment says `didn't read anything` but code belongs to `quic_write_nonblocking_epw()` function. Also is `SSL_ERROR_WANT_READ` at line 2585 the error code we want to return?

Reply:
Good catch! Thanks.